OnlineFlightSearchAPITestCases/Mocks/AirportServiceMock.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Moq;
+using OnlineFlightSearchAPI.FlightServices;
+using OnlineFlightSearchAPI.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnlineFlightSearchAPI.UnitTests.Mocks
+{
+    class AirportServiceMock : Mock<IAirportServices>
```

Comment:
access modifier?

Reply:
Removed the Separate Mocking Class as specified